docs/templates/.frogbot/frogbot-config.yml

```diff
@@ -29,6 +29,10 @@
       # The template can optionally include the ${IMPACTED_PACKAGE} and ${FIX_VERSION} variables.
       # pullRequestTitleTemplate: "[🐸 Frogbot] Upgrade ${IMPACTED_PACKAGE} to to ${FIX_VERSION}"
 
+      # [Default: false]
+      # Frogbot will aggregate all vulnerabilities fixes to one pull request at any given time
+      # aggregatePullRequests: false
```

Comment:
I recomment changing to - 
```
      # [Default: false]
      # If true, Frogbot creates a single pull request with all the fixes.
      $ If fase, Frogbot creates a separate pull request for each fix.
      # aggregateFixes: false
```
Let's also add this variable (JF_AGGREGATE_FIXES) to all workflow docs.